### ISSUE TITLE
Consider joining forces with GoogleCloudPlatform/cloud-build-notifiers

### ISSUE THREAD

**Issue by steren** [jamesward/cloud-build-notifier#1]
Not sure you saw https://github.com/GoogleCloudPlatform/cloud-build-notifiers

There are an official set of Cloud Build notifiers, notably, there is an [SMTP one](https://github.com/GoogleCloudPlatform/cloud-build-notifiers/tree/master/smtp), including [official docs](https://cloud.google.com/cloud-build/docs/configuring-notifications/configure-smtp)

**Comment by jamesward** [comment]
Yeah, saw those. I was experimenting with feeding the notifications to Cloud Monitoring alerts so I could reuse that notification infrastructure. Ran into a few issues going that route. But still investigating some options for not having to provide an smtp server.
